docs/generate_doc.py

```diff
@@ -227,8 +228,177 @@ def num2str(num):
                     f.write("\n")
 
 
+def generate_rf_material_library_doc():
```

Comment:
is this mostly copied from the `genereate_material_library_doc`? can we write in a way that we dont need to re-implement this in the future?

Reply:
Yeah, mostly from generate_material_library_doc. I'll keep this in mind and reduce the redundancy later.